src/codemodder/cli.py

```diff
@@ -43,7 +43,7 @@ class CsvListAction(argparse.Action):
     argparse Action to convert "a,b,c" into ["a", "b", "c"]
     """
 
-    def __call__(self, parser, namespace, values: str, option_string=None):
+    def __call__(self, parser, namespace, values, option_string=None):
```

Comment:
I think this was mypy, right? Isn't it weird that we fix a typing problem by ... removing typing?

Reply:
Yeah I (wrongly) introduced this on a previous PR when `mypy` wasn't working for me. The underlying code itself is not typed correctly and I thought adding this would fix it (but it does not).